service/src/main/java/com/ge/predix/acs/config/GraphConfig.java

```diff
@@ -93,17 +96,27 @@ public Graph createGraph() throws InterruptedException {
         } else {
             newGraph = TitanFactory.build().set("storage.backend", "inmemory").open();
         }
+        createSchemaElements(newGraph);
+        LOGGER.info("Initialized titan db.");
+        return newGraph;
+    }
 
+    private void createSchemaElements(final Graph newGraph) throws InterruptedException {
         createVertexLabel(newGraph, GraphResourceRepository.RESOURCE_LABEL);
         createVertexLabel(newGraph, GraphSubjectRepository.SUBJECT_LABEL);
+        createVertexLabel(newGraph, GraphGenericRepository.VERSION_VERTEX_LABEL);
+
         createIndex(newGraph, BY_ZONE_INDEX_NAME, ZONE_ID_KEY);
-        createTwoKeyUniqueCompositeIndex(newGraph, BY_ZONE_AND_RESOURCE_UNIQUE_INDEX_NAME, ZONE_ID_KEY,
-                RESOURCE_ID_KEY);
-        createTwoKeyUniqueCompositeIndex(newGraph, BY_ZONE_AND_SUBJECT_UNIQUE_INDEX_NAME, ZONE_ID_KEY, SUBJECT_ID_KEY);
-        createEdgeIndex(newGraph, BY_SCOPE_INDEX_NAME, PARENT_EDGE_LABEL, SCOPE_PROPERTY_KEY);
+        createUniqueCompositeIndex(newGraph, BY_ZONE_AND_RESOURCE_UNIQUE_INDEX_NAME, new String[] {ZONE_ID_KEY,
+                RESOURCE_ID_KEY});
 
-        LOGGER.info("Initialized titan db.");
-        return newGraph;
+        createUniqueCompositeIndex(newGraph, BY_ZONE_AND_SUBJECT_UNIQUE_INDEX_NAME,
+                new String[] { ZONE_ID_KEY, SUBJECT_ID_KEY });
```

Comment:
nitpick: why is the formatting of this call to createUniqueCompositeIndex different than the one before? Are we using the eclipse formatting rules for this project?